broker/providers/ansible_tower.py

```diff
@@ -434,6 +446,11 @@ def construct_host(self, provider_params, host_classes, **kwargs):
                 if "tower_inventory" in job_attrs
                 else self._translate_inventory(job.summary_fields.inventory)
             }
+
+            # FIXME Find better way to return ip / ansible_host from job
+            misc_attrs["ip"] = kwargs["ip"]
+            kwargs.pop("ip")
```

Comment:
can do it on a single line with a safety if the ip isn't there
```suggestion
            misc_attrs["ip"] = kwargs.pop("ip", None)
```